@xen-orchestra/web/src/composables/xo-task-properties-utils.composable.ts

```diff
@@ -0,0 +1,26 @@
+import type { XoTask } from '@vates/types'
+import { computed, type MaybeRefOrGetter, toValue } from 'vue'
+
+export function useTaskPropertiesUtils(rawTask: MaybeRefOrGetter<XoTask>) {
+  const task = computed(() => toValue(rawTask))
```

Comment:
I think you can use `toComputed` here:

```suggestion
  const task = toComputed(rawTask)
```